components/tikv_util/src/error_code/engine.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
+
+define_error_codes!(
+    "KV:Engine:",
+
+    ENGINE => ("Engine", "", ""),
+    NOT_IN_RANGE => ("NotInRange", "", ""),
+    PROTOBUF => ("Protobuf", "", ""),
+    IO => ("IO", "", ""),
+    CFNAME => ("CFName", "", ""),
+    CODEC => ("Codec", "", ""),
+
+    UNDETERMINED => ("Undetermined", "", "")
```

Comment:
I'm worried if this "undetermined error" will be messed up with transaction's undetermined error (when there is a network error during committing the primary key).